llm-guard/app/scanner_types.py

```
@@ -41,14 +41,14 @@ def __init__(self, **kwargs):
-            threshold (float): Block threshold [0,1]. Defaults to config.THRESHOLD.
+            model (str): HuggingFace model ID. Defaults to config.DEFAULT_MODEL.
+            injection_label (str): Positive-class label. Defaults to config.DEFAULT_INJECTION_LABEL.
+            threshold (float): Block threshold [0,1]. Defaults to config.DEFAULT_THRESHOLD.
             match_type (str): ``"full"`` or ``"sentence"``. Defaults to ``"full"``.
             model_max_length (int): Max token length. Defaults to 512.
         """
-        self._model = kwargs.get("model", "") or config.MODEL
-        self._injection_label = kwargs.get("injection_label", "") or config.INJECTION_LABEL
+        self._model = kwargs.get("model", "") or config.DEFAULT_MODEL
+        self._injection_label = kwargs.get("injection_label", "") or config.DEFAULT_INJECTION_LABEL
         threshold = kwargs.get("threshold", None)
```

Ruff (0.15.14)

[warning] Use `kwargs.get("threshold")` instead of `kwargs.get("threshold", None)`

Replace `kwargs.get("threshold", None)` with `kwargs.get("threshold")`

(SIM910)